airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -56,7 +59,7 @@ def jenkins_request_with_headers(jenkins_server, req):
         # Jenkins's funky authentication means its nigh impossible to distinguish errors.
         if e.code in [401, 403, 500]:
             raise JenkinsException(
-                'Error in request. Possibly authentication failed [%s]: %s' % (e.code, e.msg)
+                'Error in request. Possibly authentication failed [%s]: %s' % (e.code, e.msg)  # type: ignore
```

Comment:
This one threw an error about `msg` not being an attribute of `HTTPError`. How would `mypy` know that?

Reply:
Please try using `e.reason`:
```
    @property
    def reason(self):
        return self.msg
```

Reply:
Awesome, that looks like it worked!